### DISCUSSION TITLE
equivalent of "more" on Windows?

### DISCUSSION THREAD

**Discussion by xoofx** [nushell/nushell#4231] [Q&A] [answered]
Hey,
Love nu so far, congrats!
Trying to page pause `ls` in a directory with lots of entries. In a cmd batch I could issue `dir | more` but is there any equivalent in nushell? It seems that piping to `less` on Windows is making the output not pageable anyway (and it displays a non escapable `(END)` that force me to kill the shell entirely)
Any ideas?

**Comment by kubouch** [comment, marked as answer]
Hi! 
Glad you like Nushell!

I think in the way the output is drawn does not really play with external pagers right now. We used to have one built in but there was nobody to maintain it and it bit-rot away. Luckily, we're building a new engine that will let you pipe the output to an external pager like "less" or "more" out of the box! (You can check it at https://github.com/nushell/engine-q)
